pkg/executor/content/fetcher.go

```diff
@@ -187,3 +189,37 @@ func (f Fetcher) configureUseOfCertificate() error {
 
 	return nil
 }
+
+// CalculateGitContentType returns the type of the git test source
+func (f Fetcher) CalculateGitContentType(repo testkube.Repository) (string, error) {
+	if repo.Uri == "" || repo.Path == "" {
+		return "", errors.New("repository uri and path should be populated")
+	}
+
+	dir, err := os.MkdirTemp("/tmp", "temp-git-files")
```

Comment:
os.GetTemp()

Reply:
updated to empty string and let os.MkdirTemp(..) call this function